### PR TITLE
Request Mail fixed

### PR DESCRIPTION
**Pull request by SarahhhC**
* Contents fixed
* Yoda to BridgeMe

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ export function localSignUp(req, res, next) {
     })
     .then((existingUser) => {
       if (existingUser) {
-        res.status(201).json({ msg: userCallback.ERR_EXISTING_EMAIL });
+        res.status(201).json(existingUser);
```
Comment (sv-yoda) [inline src/server/controllers/users.controller.js]: Detected existing user when sign up, isn't there any err msg?

Comment (lyw0149) [inline src/server/controllers/users.controller.js]: It's me.

Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: That should be changed. Thanks!

File: src/server/config/json/mail.strings.js [hunk 2]
```diff
@@ -3,9 +3,11 @@
  */
 
 export default {
-  REQUEST_HTML: '<h1>Hi!</br>New mentee needs your mentoring.</h1>',
-  REQUEST_SUBJECT: 'New mentee needs your help!',
-  YODA_ACCOUNT: '"Yod Service Team" <yoda.mentor.lab@gmail.com>',
+  REQUEST_TITLE: '<h3>Hello there!</h3></br></br><h4>',
+  REQUEST_BODY: ' has requested a connection to you!</h4>',
+  REQUEST_CONTENTS: '<h4>Please check this request on BridgeMe app.</h4>',
+  REQUEST_SUBJECT: 'You’ve got a new request connection.',
```
Comment (KyuaKwon) [inline src/server/config/json/mail.strings.js]: How about  ```new request connection.``` ▶️  ```new connection request``` ?

**Comment by lyw0149** [comment]
LGTM

**Pull request merged**